benchmark_utils/transformation.py

```diff
@@ -43,10 +42,11 @@ def channels_dropout(
         The labels.
 
     """
-    transform = ChannelsDropout(probability=probability, random_state=seed)
+
     X_augm = to_numpy(X)
     y_augm = y
     for i in range(n_augmentation):
+        transform = ChannelsDropout(probability=probability)
```

Comment:
why not instantiate this object outside the loop?
Also, a seed should be given otherwise the benchmark is not reproducible.
So not sure why you made this change, can you comment?

Reply:
I put it inside the loop because when it was outside the augmentation was always giving the same X_tr

Reply:
and I have the same issue when I fix the seed

Reply:
Maybe we could predefine a sequence a seed, to have a different seed for each transformation, do you have an other idea ?

Reply:
Okay I think I know what is the exact issue, what defines the transformation is mask_len_samples and mask_start_per_sample. They are generated "randomly" in transform.get_augmentation_params, the issue is that to choose randomly, the transformation uses rng.uniform with a seed that is fixed. So we are always having the same parameters for the augmentation and we get at each iteration the same augmented data.